CITATION.rst

```diff
@@ -1,6 +1,9 @@
 Acknowledging and Citing PlasmaPy
 =================================
 
+If you use PlasmaPy for a project resulting in a publication, we
+ask that you cite the version of the software package
```

Comment:
Looks like this sentence is missing its ending?

Reply:
Oops!  I didn't mean to commit this file.  I used PyCharm's interface instead of the command line like I usually do and got it mixed up.